src/mscorlib/model.xml

```diff
@@ -6703,6 +6712,8 @@
       <Member Name="AddRef(System.IntPtr)" />
       <Member Name="AllocHGlobal(System.Int32)" />
       <Member Name="AllocCoTaskMem(System.Int32)" />
+      <Member Name="BindToMoniker(System.String)" />
+      <Member Name="ChangeWrapperHandleStrength(System.Object,System.Boolean)" />
```

Comment:
These were included because they were in the intersection of .NET 4.6.1 and Xamarin.ios